_posts/2024-07-23-geoserver-2-25-2-released.md

```diff
@@ -0,0 +1,107 @@
+---
+author: Jody Garnett
+date: 2024-07-23
+layout: post
+title: GeoServer 2.25.2 Release
+categories:
+- Announcements
+- Vulnerability
+tags:
+- Release
+release: release_225
+version: 2.25.2
+jira_version: 16923
+--- 
+
+GeoServer [2.25.2](/release/2.25.2/) release is now available
+with downloads
+([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.2/geoserver-2.25.2-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.2/geoserver-2.25.2-war.zip/download),
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.2/GeoServer-2.25.2-winsetup.exe/download)), along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.2/geoserver-2.25.2-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.2/extensions/).
+
+This is a stable release of GeoServer recommended for production use. This release is made ahead of schedule to address an urgent bug or security vulnerability.
+GeoServer 2.25.2 is made in conjunction with GeoTools 31.2, and GeoWebCache 1.25.2. 
+
+Thanks to Jody Garnett (GeoCat) for making this release on behalf of GeoCat customers. If you are not in a position to upgrade at this time please consulting the security considerations below for available mitigation measures.
+
+## Security Considerations
+
+This release addresses security vulnerabilities and is considered an essential upgrade for production systems.
+
+* [CVE-2024-36401](https://github.com/geoserver/geoserver/security/advisories/GHSA-6jj6-gm7p-fcvv)
```

Comment:
We should check if there is more in position to listed here.

I am considering listing CVEs upfront, and then they can be disclosed on their own timeframe? I am assuming end of the month is fine here.